torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2148,6 +2189,13 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=[torch.float16, torch.bfloat16]),
            ),
            sample_inputs_func=sample_inputs_index_select),
+    OpInfo('sort',
+           dtypes=all_types_and(torch.bool, torch.float16),
+           # sort on CUDA is still in the TH, no torch.bool/torch.float16 support yet
+           dtypesIfCUDA=all_types_and(torch.float16),
```

Comment:
Please add this as well after `dtypesIfCUDA`
dtypesIfRCOM=all_types_and(torch.float16)